package.go

```diff
@@ -38,6 +43,7 @@ type pkg struct {
 
 func (p *pkg) ProtoName() Name    { return Name(p.fd.GetPackage()) }
 func (p *pkg) GoName() Name       { return Name(p.name) }
+func (p *pkg) JavaName() Name     { return Name(p.fd.GetOptions().GetJavaPackage()) }
```

Comment:
This should probably return the default package name if `p.fd.GetOptions().GetJavaPackage()` returns zero  -- WDYT ?